src/makeCalculator.js

```diff
@@ -38,6 +38,33 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, b) {
+      callback(b);
+
+      return this;
+    },
+    add: (b) => {
+      calculator.result += b;
```

Comment:
```suggestion
      this.result += b;
```
Make it work with this